src/spartan/polys/univariate.rs

```diff
@@ -0,0 +1,186 @@
+//! Main components:
+//! - UniPoly: an univariate dense polynomial in coefficient form (big endian),
+//! - CompressedUniPoly: a univariate dense polynomial, compressed (omitted linear term), in coefficient form (little endian),
+use ff::PrimeField;
+use rayon::prelude::{IntoParallelIterator, ParallelIterator};
+use serde::{Deserialize, Serialize};
+
+use crate::traits::{Group, TranscriptReprTrait};
+
+// ax^2 + bx + c stored as vec![c, b, a]
+// ax^3 + bx^2 + cx + d stored as vec![d, c, b, a]
```

Comment:
Is this change just a correction for previously incorrect documentation (as mentioned elsewhere)?